index.html

```diff
@@ -4,27 +4,38 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Portfolio website</title>
-    <link href="https://fonts.google.com/specimen/Roboto" rel="stylesheet">
-    <link href="https://fonts.google.com/specimen/Poppins" rel="stylesheet">
-    <link href="https://fonts.google.com/specimen/Inter" rel="stylesheet">
-    <link href="https://fonts.google.com/specimen/Crete+Round?query=crete" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+    <link rel="stylesheet" href='https://fonts.googleapis.com/css?family=Crete Round'>
     <link rel="stylesheet" href="assests/css/mobile.css">
+    <link rel="stylesheet" href="assests/css/desktop.css">
   </head>
   <body>
     <header class="nav-bar">
         <a href="#">Chere Lemma</a>
         <nav class="navigation">
           <img src="assests/images/normal_button.png" alt="Normal Button">
         </nav>
+        <nav class="top-nav">
+          <ul>
+            <li>Portoflio</li>
+            <li>About</li>
+            <li>Contact</li>
+          </ul>
+          <img src="assests/images/icon_mail.png" alt="mail icon">
+        </nav>
     </header>
 
     <main>
-      <section class="headline">
-        <h1 class="primary-text">Hey There. I&rsquo;m Chere</h1>
-        <h2 class="support-text">I&rsquo;am a Software Developer</h2>
-  
-        <p class="description">I can help you build a product, feature or website Look through some of my work and experience! 
-          If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <section class="headline-container">
+        <article class="headline">
+          <h1 class="primary-text">Hey There. <br> I&rsquo;m Chere</h1>
+          <h2 class="support-text">I&rsquo;am a Software Developer</h2>
+    
+          <p class="description">I can help you build a product, feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, <br> don’t hestiate to contact me.</p>
+        </article>
         
         <ul class="social-icons">
```

Comment:
- [ ] You should stick to the design as much as possible (e.g., font, colors, images, text, margins). Please fix the spacing as there is the white spacing on the left and right of the page and if you see the design then you will see the background is covering the whole width. please see the image below for reference.
- [ ] If you see the design then the text is aligned to the left but it doesn't pass the middle of the page. so I would suggest you please fix this issue as well. 😃 
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/199891367-013eb82e-fae6-4899-8a2a-8a700e119a30.png)
### Your design
![his](https://user-images.githubusercontent.com/81727988/199891393-02e4fa40-e90d-4a34-95e4-f7c266226275.png)